### ISSUE TITLE
Translation off by one verse for Job 16 from verse 4

### ISSUE THREAD

**Issue by amir-zeldes** [CopticScriptorium/corpora#101]
The Coptic versification is different and creates an off-by-one error, we should fix this and re-release the OT corpus when we have a chance.

**Comment by amir-zeldes** [comment]
Also broken: Joshua 9. Comment from L. Rafferty: 

The translation given is for Joshua 9:2. The translation may be broken in this document because of verse numbering differences, e.g. in Brenton vv. 30-35 come between v. 2 and v. 3. Other editions number those verses 2a-2f, or renumber all of the verses sequentially 1-33. In Brenton this is Joshua 9:12: These are the loaves—we took them hot for our journey on the day on which we came out to come to you; and now they are dried and become mouldy.

**Comment by ctschroeder** [comment]
There is a notes file and mapping file in the [OT directory of the bible-dev repo](https://github.com/CopticScriptorium/bible-dev/tree/master/OT) that should be consulted when processing the Coptic OT. If these errors are not already in the mapping file we should modify that file

**Comment by amir-zeldes** [comment]
Yeah, that's already implemented, but Job is not mentioned there (search for b18); these issues can indeed be added there.

**Comment by amir-zeldes** [comment]
Another one from L. Rafferty: Translation is off by one verse, "with the edge of the sword" corresponds to ϩⲛⲟⲩϩⲱⲧⲃⲛⲥⲏϥⲉ in the next verse in Deuteronomy 13.

**Comment by amir-zeldes** [comment]
Note to self: Job 16 is fixed upstream, will propagate on next NLP re-run.

**Comment by amir-zeldes** [comment]
OK, these should all be done, we can close this whenever OT is re-released.

**Comment by ctschroeder** [comment]
OT rereleased in 2025 so closing
